packages/unit-testing/stubs/config/config.js

```diff
@@ -0,0 +1,52 @@
+define([], function () {
+    /**
+     * SuiteScript module
+     *
+     * @module N/config
+     * @NApiVersion 2.x
+     */
+    var config = function () { };
+
+    /**
+     * Load a configuration object with a specific type
+     * @governance 10 units
+     * @restriction Server SuiteScript only
+     *
+     * @param {Object} options
+     * @param {string} options.type one of the Type values
+     * @param {boolean} options.isDynamic load record in dynamic or deferred dynamic mode
+     * @return {Record}
```

Comment:
Should we inject RecordInstance?


Like:
```
define(['../record/RecordInstance], function(Record) {
    ...
})

Reply:
Yep